assets/css/main.css

```diff
@@ -290,6 +291,7 @@ a:hover .feather-sun {
   display: block;
   padding: 6px 12px;
   text-decoration: none;
+  
```

Comment:
remove this empty line

Reply:
In your code what does this line show then ?

Reply:
```css
.post-tag a {
  color: inherit;
  display: block;
  padding: 6px 12px;
  text-decoration: none;
}
```